src/plugins/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -352,8 +352,13 @@ export class SearchInterceptor {
             );
           }),
           tap((response) => {
-            if (this.deps.session.isRestore() && response.isRestored === false) {
-              this.showRestoreWarning(this.deps.session.getSessionId());
+            const isSearchInScopeOfSession = sessionId === this.deps.session.getSessionId();
```

Comment:
should we make it?

```suggestion
            const isSearchInScopeOfSession = sessionId  && sessionId === this.deps.session.getSessionId();
```

for the requests which aren't part of any session